main/src/cgeo/geocaching/network/StatusUpdater.java

```diff
@@ -79,10 +97,14 @@ public void run() {
                 } else {
                     gcMembershipParameters = null;
                 }
+
+                final Parameters activeConnectorsParameters = new Parameters("active_connectors",
+                        getActiveConnectorsString());
+
                 Network.requestJSON("https://status.cgeo.org/api/status.json",
                         Parameters.merge(new Parameters("version_code", String.valueOf(Version.getVersionCode(app)),
                                 "version_name", Version.getVersionName(app),
```

Comment:
You could insert "active_connectors" and `getActiveConnectorString()` directly without using an intermediate variable. Those intermediate variables are introduced only when we don't know for sure that the parameter will be present (for example the installer can be absent).

Reply:
done